### PR TITLE
[1.11] Add disable toggle to freerunners.

### PR DESCRIPTION
**Pull request by thommy101**
Adds feature #3808.

Also changed the default armor switch key binding from F to G as F is currently a default key binding for switching held item with offhand.

### CODE REVIEW

File: src/main/java/mekanism/client/MekanismKeyHandler.java [hunk 1]
```diff
@@ -46,17 +48,19 @@
 {
 	public static final String keybindCategory = "Mekanism";
 	public static KeyBinding modeSwitchKey = new KeyBinding("Mekanism " + LangUtils.localize("key.mode"), Keyboard.KEY_M, keybindCategory);
-	public static KeyBinding armorModeSwitchKey = new KeyBinding("Mekanism " + LangUtils.localize("key.armorMode"), Keyboard.KEY_F, keybindCategory);
+	public static KeyBinding armorModeSwitchKey = new KeyBinding("Mekanism " + LangUtils.localize("key.armorMode"), Keyboard.KEY_G, keybindCategory);
```
Comment (thiakil) [inline src/main/java/mekanism/client/MekanismKeyHandler.java]: Why the change to the existing keybind default? Too many clashes?

Comment (thommy101) [inline src/main/java/mekanism/client/MekanismKeyHandler.java]: See original comment ;) Minecraft uses F for switching items with offhand

Comment (thiakil) [inline src/main/java/mekanism/client/MekanismKeyHandler.java]: -facepalm- my bad! It's late xD

Comment (thommy101) [inline src/main/java/mekanism/client/MekanismKeyHandler.java]: Don't worry, we all do make comments like that :)

**Comment by aidancbrady** [comment]
Wow! Impressive, you followed the jetpack/gas tank implementation and added full client-server sync for other users. This would make it easy to add some kind of graphic change to the free runners in the future when the mode is changed. Thanks!
